__tests__/app.test.js

```diff
@@ -52,6 +52,22 @@ describe('GET requests', () => {
         })
     })
 
+    describe("GET /api/users", () => {
+        test.only('200 - should respond with an array of user objects', () => {
+            return request(app)
+            .get("/api/users")
+            .expect(200)
+            .then(({body}) => {
+                expect(body.users).toHaveLength(4)
+                body.users.forEach((user) => {
+                    expect(user).toHaveProperty("username", expect.any(String))
+                    expect(user).toHaveProperty("name", expect.any(String))
+                    expect(user).toHaveProperty("avatar_url", expect.any(String))
+                }) 
+            })
+        });
+    })
+
```

Comment:
Hi, I couldn't think of any errors to test for. I was thinking they either put in the right path or they don't